src/ai/model/Model.py

```diff
@@ -1,3 +1,97 @@
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+import tensorflow as tf
+import h5py
+from ai.exceptions.model import *
+
 class Model():
-    def __init__(self):
-        pass
\ No newline at end of file
+
+    # Model values
+    __model = None
+    __layers = []
+    __history = None
+    __compiled = False
+
+    # Class parameters and constants
+    __has_file = False
+    __activations = ["relu", "softmax", "tanh", "sigmoid", "linear"]
+    __file_name = None
+    __path = None
+    __save = False
+
+    def __init__(self, file = None, save = False, name = None, path = None):
+        if file != None:
+            self.__has_file = True
+            self.__model = tf.keras.models.load_model(file)
+        else:
+            self.__model = tf.keras.models.Sequential()
+        if save:
+            self.__save = save
+            self.__file_name = name
+            self.__path = path
+            self.__compiled = True
+
+    def add_dense_layer(self, neurons, activation):
+        if not self.__has_file:
+            if activation in self.__activations:
+                if len(self.__layers) > 0:
+                    if self.__layers[-1]['neurons'] <= neurons:
```

Comment:
File "src/main.py", line 15, in <module>
    model.add_dense_layer(500, "relu")
  File "/Users/josefernan/monest-eco/AI-dev/monest-ai/src/ai/model/Model.py", line 38, in add_dense_layer
    if self.__layers[-1]['neurons'] <= neurons:
KeyError: 'neurons'